README.md

```diff
@@ -268,22 +268,23 @@ This will launch the Cypress Test Runner UI in the `console` package, where you
 
 #### Execute Cypress in different packages
 
-An alternate way to execute cypress tests is via [test-cypress.sh](test-cypress.sh) which takes a `-p <package>` parameter to allow execution in different packages. It also can run Cypress tests in the Test Runner UI or in `-- headless` mode:
+An alternate way to execute cypress tests is via [frontend/integration-tests/test-cypress.sh](frontend/integration-tests/test-cypress.sh) which takes a `-p <package>` parameter to allow execution in different packages. It also can run Cypress tests in the Test Runner UI or in `-- headless` mode:
 
 ```
-console>./test-cypress.sh
+frontend > ./integration-tests/test-cypress.sh
```

Comment:
@jerolimov Can it be modified in this way 
```suggestion
console> frontend > ./integration-tests/test-cypress.sh
```

Reply:
@sanketpathak I changed it to

```
console/frontend > ./integration-tests/test-cypress.sh
```

ptal?